data/governments.txt

```diff
@@ -486,6 +486,7 @@ government "Hai (Unfettered)"
 	
 	"player reputation" -1000
 	"attitude toward"
+		"Hai (Unfettered) (Wanderer Tribute)" 1
```

Comment:
```suggestion
		"Hai (Unfettered) (Wanderer Tribute)" 2
```
These are ships that they cant produce, carrying food which they need and equipped with jump drives.
They should care more